src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -33,12 +33,25 @@ public partial class ScriptEditorWindow : ModelessChildWindow
         public PythonNode nodeModel { get; set; }
         private bool nodeWasModified = false;
         private string originalScript;
-        public PythonEngineVersion CachedEngine { get; set; }
+
+        public string CachedEngineName { get; set; }
```

Comment:
because this is a view - I would say let's just break these public signatures.

Reply:
well...that would simplify some of the changes. Will do